frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -56,6 +56,7 @@ export function MissionHeader({ mission }: MissionHeaderProps) {
             <StatusReason mission={mission}></StatusReason>
             <InfoSection>
                 <MissionStatusDisplay status={mission.status} />
+                {HeaderText(TranslateText('Date') + ': ' + startDate)}
```

Comment:
Would either rename from "Date" to "Start date" or include date as a part of "Start time"

In case a mission goes passed midnight